tesseract/tesseract_scene_graph/test/CMakeLists.txt

```diff
@@ -11,3 +11,13 @@ target_code_coverage(${PROJECT_NAME}_unit ALL EXCLUDE ${COVERAGE_EXCLUDE} ENABLE
 add_gtest_discover_tests(${PROJECT_NAME}_unit)
 add_dependencies(${PROJECT_NAME}_unit ${PROJECT_NAME})
 add_dependencies(run_tests ${PROJECT_NAME}_unit)
+
+add_executable(${PROJECT_NAME}_utils_unit utils_unit.cpp)
+target_link_libraries(${PROJECT_NAME}_utils_unit PRIVATE GTest::GTest GTest::Main ${PROJECT_NAME} tesseract::tesseract_support tesseract::tesseract_geometry)
+tesseract_target_compile_options(${PROJECT_NAME}_utils_unit PRIVATE)
+tesseract_clang_tidy(${PROJECT_NAME}_utils_unit)
+tesseract_code_coverage(${PROJECT_NAME}_utils_unit ALL EXCLUDE ${COVERAGE_EXCLUDE})
+target_include_directories(${PROJECT_NAME}_utils_unit PRIVATE ${orocos_kdl_INCLUDE_DIRS})
```

Comment:
The above macros have changed to leveraging the one in cmake_common_scripts. I would take a look at another package for the new syntax.

Reply:
> > I think every time a pair is added or removed it would update the information. This will not affect the lookup during normal operation so should not see a penalty. Only would affect add/removal of objects. Do you see a case where we would be constantly adding and remove objects from the allowed collision matrix? If so, I would say we leave the way you have it.
> 
> Well my usecase is that every time I plan I need to add a link and a bunch of things to the ACM. However the environment I use is a clone that isn't reused. So in that case it would just be wasted overhead. Perhaps I can add a getter that on the first time runs the utility and stores the results. Then I can have a bool to keep track of when it needs to be updated. Essentially, just cache the results.



> > I think every time a pair is added or removed it would update the information. This will not affect the lookup during normal operation so should not see a penalty. Only would affect add/removal of objects. Do you see a case where we would be constantly adding and remove objects from the allowed collision matrix? If so, I would say we leave the way you have it.
> 
> Well my usecase is that every time I plan I need to add a link and a bunch of things to the ACM. However the environment I use is a clone that isn't reused. So in that case it would just be wasted overhead. Perhaps I can add a getter that on the first time runs the utility and stores the results. Then I can have a bool to keep track of when it needs to be updated. Essentially, just cache the results.

I think that is a good middle ground for now.